src/OrchardCore.Themes/TheTheme/Views/MenuItemLink-LinkMenuItem.cshtml

```diff
@@ -4,17 +4,16 @@
     ContentItem contentItem = Model.ContentItem;
     var linkMenuItemPart = contentItem.Content["LinkMenuItemPart"];
     int level = (int)Model.Level;
-}
 
-@if (level > 0)
-{ 
-    <a class="dropdown-item" href="@Url.Content((string)linkMenuItemPart.Url)">@linkMenuItemPart.Name</a>
-}
-else if ((bool)(Model.HasItems))
-{
-    <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="@Url.Content((string)linkMenuItemPart.Url)">@linkMenuItemPart.Name<b class="caret"></b></a>
-}
-else
-{
-    <a class="nav-link" href="@Url.Content((string)linkMenuItemPart.Url)">@linkMenuItemPart.Name</a>
+    var name = (string)linkMenuItemPart.Name;
+    if (Model.Level == 0 && Model.HasItems)
+    {
+        name += "<b class=\"caret\"></b>";
```

Comment:
Can you do an extra AppendHtml instead of concatenating the strings?

Reply:
Done.